docs/FAQ

```diff
@@ -427,6 +428,22 @@ FAQ
   have their pros and cons, and we try to maintain a comparison of them here:
   https://curl.se/docs/ssl-compared.html
 
+  2.3 How do I upgrade curl.exe in Windows?
+
+  The curl tool that is shipped as an integrated component of Windows 10 and
+  Windows 11 is managed by Microsoft. If you were to delete the file or
+  replace it with a newer version downloaded from https://curl.se/windows,
+  then Windows Update will cease to work on your system.
```

Comment:
Saying "Windows Update will cease to work on your system" seems like overkill, I'd think it would update the tool or not, I'm not sure which I've never tried it. Anyone know what happens?

Reply:
It completely stops windows update because it detects windows as having been tampered with. I think my phrasing here is correct. It will not update *anything* after you do that.

Reply:
What OS? I tried it in Windows 10 and I cannot reproduce. I resumed an old VM image running 20H2, took ownership of curl.exe and deleted it, then updated Windows. The first few updates were successful and curl.exe was not added back. Then I installed the cumulative update, also successful, and it had readded curl.exe (but a later version 7.79.1).

Reply:
I have not done this myself, but I have had multiple persons contacting me about it.

Reply:
Ok fair enough.